### DISCUSSION TITLE
Как сменить часовой пояс для отображения логов

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#463] [Как это сделать?] [answered]
Возникла проблема: поставили на сервер с UTC, потом перенастроили PHP на Europe/Moscow, но логи все равно пишутся по UTC, хотя везде стоит часовой пояс +3. Напишите, пожалуйста, каким образом это можно поправить?

**Comment by erpic-ru** [comment, marked as answer]
**Ответ**

Время для БД устанавливается в момент ее первой инициализации.

Необходимо зайти в psql и изменить часовой пояс для БД и потом рестартовать базу

```
nano /etc/postgresql/12/main/postgresql.conf
```

Найти и поправить: 

```
timezone = 'Europe/Moscow'
log_timezone = 'Europe/Moscow'
```

Перезагрузить.

Проверить:

```
su - postgres -c "psql -d totum -c 'SHOW TIME ZONE;'"
```

**Reply by ttm-alexeypolunin** [reply]
Таймзона должна быть указана валидная. Что бы ее найти используйте тот-же `tzselect`, что и при установке (ток не завершайте его, а скопируйте значение TZ и выйдите `Ctrl+C`):

<img width="699" alt="Снимок экрана 2023-10-27 в 13 03 16" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/0111e09d-cf0a-495e-a8fb-f19279047351">
